src/client/datascience/kernel-launcher/localKnownPathKernelSpecFinder.ts

```diff
@@ -68,7 +79,8 @@ export class LocalKnownPathKernelSpecFinder extends LocalKernelSpecFinderBase {
         // Find all the possible places to look for this resource
         const paths = await this.jupyterPaths.getKernelSpecRootPaths(cancelToken);
         const searchResults = await this.findKernelSpecsInPaths(paths, cancelToken);
-
+        const oldDernelSpecsDeleted = this.oldDernelSpecsDeleted;
```

Comment:
Kernel

Reply:
I like to leave a few of these easter eggs for you.